src/transformState.js

```diff
@@ -46,7 +46,25 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const value of transforms) {
+    if (value.operation === 'addProperties') {
+      for (const i in value.properties) {
+        state[i] = value.properties[i];
+      }
+    } else if (value.operation === 'removeProperties') {
+      for (const i of value.properties) {
```

Comment:
"i" - that bad name, when you use "for of" loop, better to use something more meaningful